tests/DfE.FindInformationAcademiesTrusts.Data.UnitTests/AcademyTests.cs

```diff
@@ -0,0 +1,54 @@
+using DfE.FindInformationAcademiesTrusts.Data.UnitTests.Mocks;
+
+namespace DfE.FindInformationAcademiesTrusts.Data.UnitTests;
+
+public class AcademyTests
+{
+    [Theory]
+    [InlineData(25, 100, 25)]
+    [InlineData(1, 4, 25)]
+    [InlineData(816, 1074, 76)]
+    [InlineData(100, 100, 100)]
+    [InlineData(101, 100, 101)]
+    [InlineData(0, 100, 0)]
+    public void PercentageFull_should_return_the_correct_percentage_calculation(int numberOfPupils,
+        int capacity, int expectedPercentageNumber)
```

Comment:
`PercentageFull` is a float but being tested as though it is an int.
You can test floats using Fluent Assertion's `value.Should().BeApproximately(3.14F, 0.01F);`